src/test/test-data/paymentCoordinator/processClaimProofs_Root1.json

```diff
@@ -0,0 +1,54 @@
+{
+  "Root": "0x5a17c863bab9249dd3463e6f942c51e3a885b46632c3e127666b5c6d9efbefda",
```

Comment:
When we generate the roots, we need to confirm that the library we use generates the tree the same way as openzeppelin's merkle-tree js library.  There is a note in our merkle.sol library that relates to the extra precautions OZ takes

Reply:
good callout, the leaves we use are

(address, root) = 52 bytes

and

(token address, amount = 52 bytes

Reply:
it still depends on the implementation for creating the leaves, whether we encode or encodePacked
<img width="618" alt="Screenshot 2024-04-12 at 8 50 55 AM" src="https://github.com/Layr-Labs/eigenlayer-contracts/assets/12021290/b237ca67-2cc7-4f7c-a845-0eaf184aa80f">